src/utils/tab_utils.ts

```diff
@@ -0,0 +1,21 @@
+import fs from "fs";
+import { CodeSyncLogger } from '../logger';
+import { tabEventHandler } from "../events/tab_event_handler";
+import { formatDatetime } from "./common";
+
+export const removeTabFile = (filePath: string, funcName: string) => {
+	fs.unlink(filePath, err => {
```

Comment:
We should return if filePath does not exist, either i nthis funciton or before this function is called

Reply:
I suggested to check if path exists or not, 
```if (!fs.existsSync(fielPath)) return;```

Reply:
Oh okay, thanks for clarifying